ergotree-interpreter/src/eval/global_vars.rs

```diff
@@ -68,4 +71,11 @@ mod tests {
             ctx.inputs
         );
     }
+
+    #[test]
+    fn eval_group_generator() {
+        let ctx = Rc::new(force_any_val::<Context>());
+        let res = eval_out::<EcPoint>(&GlobalVars::GroupGenerator.into(), ctx.clone());
+        assert!(matches!(res.into(), Value::GroupElement(_)));
```

Comment:
For regression test checking that evaluation returns an exact value (`dlog_group::generator()`) would be a better fit.

Reply:
Fixed.